composer.json

```diff
@@ -9,12 +9,6 @@
             "email": "hi@metadrop.net"
         }
     ],
-    "repositories": [
-      {
-        "type": "vcs",
-        "url": "https://github.com/Metadrop/drupal-behat.git"
-      }
-    ],
     "require": {
       "nuvoleweb/drupal-behat": "^1.2",
```

Comment:
The library has released a new 1.3 version, I guess we should require at least that release, right?

Reply:
```suggestion
      "nuvoleweb/drupal-behat": "^1.3",
```